qubesadmin/tools/qvm_start_daemon.py

```diff
@@ -278,7 +278,11 @@ def get_monitor_layout():
             ['xrandr', '-q'], stdout=subprocess.PIPE).stdout:
         line = line.decode()
         if not line.startswith("Screen") and not line.startswith(" "):
-            output_params = REGEX_OUTPUT.match(line).groupdict()
+            match = REGEX_OUTPUT.match(line)
+            if not match:
+                warnings.warn('Invalid output from xrandr: {!r}'.format(line))
```

Comment:
First of all `warnings` is not imported here. But then, I'd recommend using `logging` module instead.